pkgs/top-level/python-packages.nix

```diff
@@ -7119,6 +7119,11 @@ self: super: with self; {
 
   mutf8 = callPackage ../development/python-modules/mutf8 { };
 
+  mujoco = callPackage ../development/python-modules/mujoco {
+    inherit (pkgs) mujoco;
+    pythonVersion = python.version;
```

Comment:
Ah I led you slightly astray. I was under the impression that `pythonVersion` was automatically passed as an argument to python module derivations, but it lives as `python.pythonVersion`. see eg [here](https://github.com/NixOS/nixpkgs/blob/67c0a676dd5e9998beb12d4f79ec44d676181802/pkgs/development/python-modules/torchvision/bin.nix#L16)